config/locales/en.yml

```diff
@@ -440,8 +441,8 @@ en:
           true: Allow requests
           false: Do not allow requests
         availability_preference_fixed:
-          true_html: Display specific dates
-          false_html: Display a description of when you'll host school experience candidates
+          true_html: Show specific dates
```

Comment:
Same as above, the old labels (that should have been removed in a prior PR) were showing now I reverted to using the form builder, the text has since changed so it was updated in the locales file